app/controllers/users/registrations_controller.rb

```diff
@@ -17,7 +17,7 @@ def create
   private
 
   def configure_sign_up_params
-    params.require(:user).permit(:email, :password, :password_confirmation, :admin)
+    params.permit(:email, :password, :password_confirmation, :admin, :firstname, :lastname, :age, :avatar)
```

Comment:
@josephelias-rubyians  Its a good practise to use the require user to encapsulate the attributes

Reply:
I have used the require attribute but it fails when passing a avatar to the params. So I have changed the params likes this. And while passing these datas from postman (passing as form-data, also avatar as of type file) the params are passing like this {"":"", "":"", ......} .

Reply:
data[avatar]
data[media_asset_files_attributes][0][name]